frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -6,12 +6,12 @@
 <div *ngIf="!hasRoute('projects/create')" class="mr-2.5">
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/f27ae70d-d98d-49ff-bb37-504947e0f907)

The "delete" icon is not centered vertically in the button. Also, the `<div>` which contains "techuser and "Manager, read & write" is not centered vertically.